docker/prowbazel/bootstrap.py

```diff
@@ -0,0 +1,965 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
any way to reference rather than copy this large file ? or did you make changes to it (which change compared to the original ? where is the original?)